vueui/src/main.js

```diff
@@ -2,6 +2,7 @@
  * Vue.js based ui framework for SS13
  * Made for Aurora, by Karolis K.
  */
+import "core-js/stable"
```

Comment:
wot is this.

Reply:
It's for `.repeat()`, it's a shim/compatibility library. Karolis suggested I include it instead of writing my own shim.